recipes/cad1/task1/baseline/evaluate.py

```diff
@@ -195,13 +196,15 @@ def _evaluate_song_listener(
             np.array(listener_audiograms["audiogram_levels_l"]),
             np.array(listener_audiograms["audiogram_cfs"]),
             config.nalr.fs,
+            65 - 10 * np.log10(compute_rms(left_reference_signal)),
         )
         per_instrument_score[f"right_{instrument}"] = compute_haaqi(
             right_enhanced_signal,
             right_reference_signal,
             np.array(listener_audiograms["audiogram_levels_r"]),
             np.array(listener_audiograms["audiogram_cfs"]),
             config.nalr.fs,
+            65 - 10 * np.log10(compute_rms(right_reference_signal)),
```

Comment:
Shouldn't this be
```
65 - 20 * np.log10(compute_rms(right_reference_signal))
```

i.e. with a factor of 20 rather than 10.
 
The rms value is an amplitude measurement rather than a power measurement.

https://dspillustrations.com/pages/posts/misc/decibel-conversion-factor-10-or-factor-20.html